Library/ViewModels/DashboardRewardsCellViewModel.swift

```diff
@@ -56,16 +56,16 @@ public final class DashboardRewardsCellViewModel: DashboardRewardsCellViewModelT
   DashboardRewardsCellViewModelInputs, DashboardRewardsCellViewModelOutputs {
   // swiftlint:disable function_body_length
   public init() {
-    let statsProject = self.statsProjectProperty.signal.ignoreNil()
+    let statsProject = self.statsProjectProperty.signal.skipNil()
 
     let rewards = statsProject
-      .map { stats, project in (project.rewards ?? [], stats) }
+      .map { stats, project in (project.rewards, stats) }
       .map(allRewardsStats(rewards:stats:))
 
     let initialSort = rewards.sort { $0.pledged > $1.pledged }
 
     let sortedByTop = rewards
-      .sort { $0.minimum > $1.minimum }
+      .sort { ($0.minimum ?? 0) > ($1.minimum ?? 0) }
```

Comment:
ATTN: swift 3 no longer has optional comparisons! which is a good thing, but now gotta do this kind of jazz...